src/tools/rados/rados.cc

```diff
@@ -3661,8 +3925,10 @@ int main(int argc, const char **argv)
       opts["target_locator"] = val;
     } else if (ceph_argparse_witharg(args, i, &val, "--target-nspace" , (char *)NULL)) {
       opts["target_nspace"] = val;
+#ifdef USE_LIBRADSSTRIPER
```

Comment:
typo? Should it be `#ifdef WITH_LIBRADOSSTRIPER` here?

Reply:
Good catch, thank you!